.changeset/ten-dryers-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-azure-devops-backend': minor
+'@backstage/plugin-azure-devops': minor
+'@backstage/plugin-azure-devops-common': patch
+---
+
+Azure DevOps plugin is now integrated with permission framework for its core features, see the https://github.com/backstage/backstage/blob/master/plugins/azure-devops/README.md for more details.
```

Comment:
```suggestion
Azure DevOps plugin is now integrated with permission framework for its core features, see the [`README`](https://github.com/backstage/backstage/blob/master/plugins/azure-devops/README.md#permission-framework) for more details.
```

Reply:
Changeset doc ci verification failed with `Links in changesets must use absolute URLs`. So changing back to 

> Azure DevOps plugin is now integrated with permission framework for its core features, see the [`README`](https://github.com/backstage/backstage/blob/master/plugins/azure-devops/README.md) for more details.